perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1789,21 +1792,31 @@ public Application approveApplicationInternal(PerunSession sess, int appId) thro
 				Utils.checkMaxLength("TitleBefore", candidate.getTitleBefore(), 40);
 				Utils.checkMaxLength("TitleAfter", candidate.getTitleAfter(), 40);
```

Comment:
You can probably take a lot of code above, starting with ```// put application data into Candidate``` comment and make it a private method, then use it only for the case when member is really created from the candidate object.

Reply:
Done. You are right, I just did not want to refactor the code too much because there would be lots of changes.